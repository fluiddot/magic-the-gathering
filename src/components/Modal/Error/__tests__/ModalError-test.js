import React from "react";
import ModalError from "../ModalError";
import renderer from "react-test-renderer";

test("renders correctly", () => {
  const tree = renderer.create(<ModalError text="Mock text" visible={true} onClose={() => {}} />).toJSON();
  expect(tree).toMatchSnapshot();
});
