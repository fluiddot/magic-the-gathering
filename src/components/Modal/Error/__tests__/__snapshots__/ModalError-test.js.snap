// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Modal
  animationType="slide"
  hardwareAccelerated={false}
  style={
    Object {
      "height": 120,
      "justifyContent": "center",
      "width": 250,
    }
  }
  transparent={false}
  visible={true}
>
  <View>
    <Text>
      Mock text
    </Text>
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={null}
    >
      <Text
        style={null}
      >
        Close
      </Text>
    </View>
  </View>
</Modal>
`;
